### ISSUE TITLE
Magus Sov Bonus not Working

### ISSUE THREAD

**Issue by Vinfrid** [WarEmu/WarBugs#20956]
And again i was try my damage on conctruct and it lower after returning in game from spring leaving
*Expected behavior and actual behavior:*
Damage with 5 pieces Main Soverein bonus must be higher
*Steps to reproduce the problem:*
I tested with 272 damage bonus (Bloodlord staff, 5sov, 3triumph and 2LotD+1LotD)  
And after test Change Sov on Invader with same damage bonus(using Liminent)
But critDamage still same 
*Testing Screenshots*
![image](https://github.com/WarEmu/WarBugs/assets/76210368/d9bc1be0-6f3f-48b5-89e6-6099807ba188)
![image](https://github.com/WarEmu/WarBugs/assets/76210368/b23d3c18-aba7-44c8-8aa8-ba1a269fcabe)

**Comment by lilimx** [comment]
Hey,

what does the "and again" mean?

**Comment by Vinfrid** [comment]
After i make Magus it was nerfed one time (-20% from turret) and broken, now 2 times(that what i mean "and again")
Its very dissapionting, how much time and effort was spent, and now we have this - class without damage(much lower than other, i tested)and without any range utility(not have Heal debuff, not have range snair skill, excluding 1 moral, not have range silence or knokdown)
(Engi here too without any damage)

**Comment by TornixDM** [comment]
i can confirm the +15% crit damage from 4 piece sov for magus does not work i have done 2 separate tests using enemy combat log statistics hitting the target dummies with all damaging abilities for hundreds of hits to ensure the crits reached their highest damage roll? (crit damage seems kind of random) one log without 
![without 15%](https://github.com/WarEmu/WarBugs/assets/139567259/02ccf26c-6d8c-4d70-944d-db6a1fee9f1f)
and this second one is with 4 piece sov
![with 15%](https://github.com/WarEmu/WarBugs/assets/139567259/3188f6ff-c3bc-406f-8164-2a1e43f47871)
if it was working the max crits would be somewhat higher

**Comment by Undrsiege** [comment]
Yes, I did some tests on this 15% crit damage set bonus and I came to the same conclusion that it is not functioning.  I tested for a good bit and took into account the variable crit bonus of 40% to 60% crit bonus ROR uses.  Using identical damage bonus with and without the set bonus...
